core/src/test/java/com/linecorp/armeria/server/VirtualHostBuilderTest.java

```diff
@@ -333,7 +333,7 @@ void precedenceOfDuplicateRoute() {
                                                                         reqTarget, RoutingStatus.OK);
         final Routed<ServiceConfig> serviceConfig = virtualHost.findServiceConfig(routingContext);
         final Route route = serviceConfig.route();
-        assertThat(route).isSameAs(routeA);
+        assertThat(route).isEqualTo(routeA);
```

Comment:
https://github.com/line/armeria/pull/4802/files#diff-cd2df9e0b65586c6cbd6c95b5cd88d4460b02e78389a5011192119104160956eR126

`contextPath = "/";` seems to create new `Route`.

Reply:
Sure, sounds fine to me